pro/src/core/OfferEducationalStock/constants.ts

```diff
@@ -0,0 +1,9 @@
+import { OfferEducationalStockFormValues } from './types'
+
+export const DEFAULT_EAC_STOCK_FORM_VALUES: OfferEducationalStockFormValues = {
+  eventDate: '',
+  eventTime: '',
+  numberOfPlaces: '',
+  totalPrice: '',
+  bookingLimitDatetime: '',
```

Comment:
du coup on ne c'était pas dis : 
```suggestion
  eventDate: '',
  eventTime: '',
  numberOfPlaces: null,
  totalPrice: null,
  bookingLimitDatetime: '',
```

et (dans mon souvenir les forms values pour les dates sont des strings, c'est seulement au moment de la serialisation qu'on renvoi une date... à checker) : du coup ça donnerai :

```ts
export type OfferEducationalStockFormValues = {
   eventDate: string
   eventTime: string
   numberOfPlaces: number | null
   totalPrice: number | null
   bookingLimitDatetime: string
 }
```

à tester ensemble